config/rbac/role.yaml

```diff
@@ -133,13 +121,3 @@ rules:
   - get
   - patch
   - update
-- apiGroups:
```

Comment:
Is there a separate set of roles for the agent that includes ingress?
Can kubebuilder be used for the agent too?

Reply:
good point. I'll make and test some now, and add it to this PR (or it could be a separate PR to keep this one moving?)

Reply:
For now, let's capture an issue for the deploy of the agent.
Priority wise, deploying it to a cluster is lower than moving on to the syncer piece